### PR TITLE
Added UX picture and edited the design doc.

### PR DESCRIPTION
**Pull request by JWWJGit**
I updated the UX picture and defined how the modules should look in the design doc.

### CODE REVIEW

**Comment by YLee2121** [comment]
@alex308248 will take care of the PR

**Comment by alex308248** [comment]
@Weirdojord
You draw a good UI UX picture that shows how UX works, and your third part also list down the functions in different files. It is much better than your first version. But there are some big issues that you need to fix in your PR.


1. This project do not include login function, it is great if you add it. However, the most important point is the usage of required functions, and you didn't add them in your picture.
2. Your UI pciture looks like a website or an mobile app, but this is a CLI project. All of your function need to be triggered by commends. Buttons should not exist in this project.
3. You didn't mention what will happen after login success. I assume there should be a page to use `number_of_days`, `number_of_leap_years` and `get_week_of_day` functions. 
4. There are something you'll need to modify in the third part. For example, don't use `=`  but use `:`, because it means assign a number to the variable. Also don't use `'''` because you are writing a design doc, not your code. You can add `'''` in your code when you implement it. I write an example for you.

```
# File1 (Please write down the file name if you are going to seperate them in different file)
login(username, password):
    username: string, password: string (string is the name of data structure, str() is only function)
    return: bool # return true if the login is successful  (If you only wnat to return the login is successfil or not, you should use a bool rather than a string)
    Purpose of this function: Check if the login information is right, and link the login action to Log module.
    

# File2 (Your file name)
number_of_days(year, month):
    year: int, month: int
    return: int
    Purpose of this function: The function will return the total days in a specific month of the year. We will build a python module leveraging _calendar.monthrange(year, month)_ to accomplish this requirement.
    
```

File: doc/design_doc.md [hunk 1]
```diff
@@ -9,52 +9,132 @@
...
-    | --------- | ---- | ------------- |
-    | DateTime1 | Mike | Command1(...) |
-    | DateTIme2 | Mike | Command2(...) |
-    | DateTime3 | Jack | Command3(...) |
+```python
+log(username, password, status):
+    '''purpose of this function: After intaking the status from either login function or register function, combine the datetime, user's name, and activities into a string, and then open the log file and write them in.
+    '''
+    username = str, password = str, status = str
+    return = str # return the result for user.
```
Comment (alex308248) [inline doc/design_doc.md]: It is not necessary to have a return value in log module. You can add it if you want to make sure the log is recorded successful, but that should be a bool not a string.

File: doc/design_doc.md [hunk 2]
```diff
@@ -9,52 +9,132 @@
...
+main():
+    '''purpose of this function: connect the other three modules and make this project work.
+    '''
+    input("Which action you wanna take?(Login or Register)")
+
+    if Login:
+        login()
+    elif Register:
+        register()
+    else:
```
Comment (alex308248) [inline doc/design_doc.md:145]: Where is the page after login. There should be a page to use calender functions.

**Review by alex308248 — DISMISSED** [review]
Don't forget the most important part. Pls add the usage of Canlendar functions in your picture, and your UI design should not include buttons.

**Comment by JWWJGit** [comment]
Hello Alex,

Thanks for the useful feedback, I'll finish them tonight.

Best,
Jordan

**Review by YLee2121 — APPROVED** [review]
approve

**Pull request merged**